nservicebus/messaging/delayed-delivery.md

```diff
@@ -5,10 +5,21 @@ tags:
 - Defer
 ---
 
-When a message is sent it does not need to be dispatched immediately. You can specify a later time for the message to be delivered. Before version 6 this was achieved by using `bus.Defer(...)` instead of `bus.Send(...)`. In version 6, this is accomplished by calling methods on an instance of `SendOptions` that is passed into `Send(...)` calls.  
+When a message is sent it does not need to be dispatched immediately. it can be specified a later time for the message to be delivered. In Version 5 and below this can be achieved by using `bus.Defer(...)` instead of `bus.Send(...)`. In Version 6, this is accomplished by calling methods on an instance of `SendOptions` that is passed into `Send(...)` calls.  
```

Comment:
> When a message is sent it does not need to be dispatched immediately. it can be specified a later time for the message to be delivered.

The message doesn't have to be dispatched immediately after sending, it can be delivered at a later time if necessary.